sass/layout/_sidebar-left.scss

```diff
@@ -23,7 +23,51 @@ $sidebarOpacityAnimationDuration: 0.15s;
     }
 }
 
+.sidebar--left-dragged {
+    cursor: col-resize;
+    //Note: Resizing is not working if transition is set
+    transition: none !important;
+
+    #resizeLine {
+        &::after {
+            background-color: var(--denim-sidebar-active-border);
```

Comment:
@matthewbirtch Are these supposed to use a theme-specific colour, or should they be based on the theme?

Reply:
No, it is not, right color variable is `--sidebar-text-active-border`